animesaturn_downloader.py

```diff
@@ -48,11 +53,26 @@ def download_resource(resource_url: str):
     if retries > DOWNLOAD_MAX_RETRIES:
       logging.error(traceback.print_exc())
     else:
-      logging.warning(f"Error while downloading {resource_url}, retrying...")
+      logging.warning(FAIL + f"Error while downloading {resource_url}, retrying..." + ENDC)
       time.sleep(SLEEP_SECONDS_ON_ERROR)
       return download_resource(resource_url)
 
 
+def is_episode_alredy_present(out_dir:str , ep_number: int) -> bool :
```

Comment:
for e in os.listdir(out_dir):
    if ep_name in e:
        return True
return False